pages/common/git-archive.md

```diff
@@ -0,0 +1,24 @@
+# git archive
+
+> Create an archive of files from a named tree.
+> More information: <https://git-scm.com/docs/git-archive>.
+
+- Create a tar archive from the contents of the latest commit on the current branch:
```

Comment:
Maybe an extra example (at the first position) saying that the output is written to the `stdout` is better.